azure/.vscode/settings.json

```diff
@@ -7,5 +7,6 @@
         "./shared/src"
     ],
     "python.analysis.autoSearchPaths": true,
-    "python.terminal.activateEnvironment": true
+    "python.terminal.activateEnvironment": true,
+    "python.languageServer": "Pylance"
```

Comment:
Since we're doing format-on-save now I want to mess with the settings a bit (max length 80 hurts my eyes) but I can do that in a followup.